third_party/winuwp_h264/H264Encoder/H264Encoder.cc

```diff
@@ -514,8 +522,8 @@ int WinUWPH264EncoderImpl::SetRates(
 
 #ifdef DYNAMIC_FPS
   // Fps changes seems to be expensive, make it granular to several frames per second.
-  if (max_frame_rate_ != new_framerate && std::abs((int)max_frame_rate_ - (int)new_framerate) > 5) {
-    max_frame_rate_ = new_framerate;
+  if (frame_rate_ != new_framerate && std::abs((int)frame_rate_ - (int)new_framerate) > 5) {
```

Comment:
>5 [](start = 88, length = 1)

Please replace with a constant.

Reply:
We talked offline with @fibann and we believe this code could be improved, for example by handling smaller changes while ignoring jittering. So we would ideally want to change it to do so (and use a constant in the process). But given the time constraint to fix this bug, and the fact we are not familiar with the code and what the implications of changing this are, we prefer not to touch that code at all right now, and defer any change to an actual improvement of that heuristic backed by actual perf data.